src/admin/assignments/views/AssignmentsOverviewAdmin.tsx

```diff
@@ -565,12 +565,12 @@ const AssignmentOverviewAdmin: FunctionComponent<RouteComponentProps & UserProps
 					? tText('admin/assignments/views/assignments-overview-admin___ja')
 					: tText('admin/assignments/views/assignments-overview-admin___nee');
 
-			case 'labels': {
+			case 'quality_labels': {
 				const labelObjects: { id: string; label: string }[] =
-					assignment?.labels?.map((label) => {
+					assignment?.quality_labels?.map(({ label, id }) => {
 						return {
-							id: label.assignment_label.id,
-							label: label.assignment_label.label || '',
+							id: `${id}`,
```

Comment:
waarom deze wrapper? Om te converteren naar een string ofzo?
Die id is een uuid, dus het is zowiezo een string
```suggestion
							id,
```

Reply:
<img width="321" alt="Screenshot 2024-02-23 at 13 05 30" src="https://github.com/viaacode/avo2-client/assets/90180417/1fb0895d-a169-4476-9046-d8810a3492e8">
Lijkt hier anders getyped te zijn